javascript/stimulus_reflex.js

```diff
@@ -120,7 +107,24 @@ const extendStimulusController = controller => {
 
       dispatchLifecycleEvent('before', element)
 
-      subscription.send(data)
+      setTimeout(() => {
+        element.reflexData = {
+          ...data,
+          args,
+          url,
+          attrs,
+          selectors,
+          permanent_attribute_name:
+            stimulusApplication.schema.reflexPermanentAttribute,
+          reflexId: reflexId,
+          params: serializeForm(element.closest('form'), {
+            hash: true,
+            empty: true
+          })
+        }
```

Comment:
We'll want to change this slightly given the previous recommended change.

```js
const { params } = element.reflexData || {}
element.reflexData = {
  ...data,
  params: { 
    ...params,
    serializeForm(element.closest('form'), {
      hash: true,
      empty: true
    })
  }  
```

Reply:
I needed to make `serializeForm` =>  `...serializeForm` since that method returns a hash.